migrations/1757326540_settings.go

```
@@ -67,6 +67,27 @@ func init() {
+			} {
+				if os.Getenv(cap.envVar) == "" {
+					continue
+				}
+				collection, err := app.FindCollectionByNameOrId("config_values")
+				if err != nil {
+					return err
+				}
+				record := core.NewRecord(collection)
+				record.Set("key", cap.key)
+				record.Set("value", os.Getenv(cap.envVar))
+				app.Save(record)
```

golangci-lint (2.12.2)

[error] Error return value of `app.Save` is not checked

(errcheck)